src/main/java/az/keytd/expensetracker/controller/MailSenderController.java

```diff
@@ -0,0 +1,21 @@
+package az.keytd.expensetracker.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import az.keytd.expensetracker.service.MailSenderService;
+
+@RestController
+@RequestMapping("mail")
+public class MailSenderController {
+    @Autowired
+    private MailSenderService mailSenderService;
+
+    @GetMapping("sendmail")
```

Comment:
"send" in enough, again pay attention what root path is, in this case it is "mail" so path to this endpoint would be "mail/send"